roles/hydra-tutorials/defaults/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+# roles/hydra-tutorials/defaults/main.yml
+
+# the solr version needs to match the default version for [cbeer/solr_wrapper](https://github.com/cbeer/solr_wrapper/blob/master/lib/solr_wrapper.rb#L10-L12) or needs to be explicity set on the local system to match the version here
+  solr_version: 6.1.0
+
```

Comment:
Should we use a different variable name, to make it clear that this is installing solr_wrapper rather than solr under tomcat for production?